### PR TITLE
NMS-16108: OpenShift Documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16108

### CODE REVIEW

File: docs/modules/deployment/pages/core/containers.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
+[[container-deployment]]
+= Containerized Deployment
+:description: Learn how to install OpenNMS Horizon/Meridian in a containerized environment like Kubernetes or Red Hat OpenShift.
```
Comment (mershad-manesh) [inline docs/modules/deployment/pages/core/containers.adoc:4]: Would it be better to say something like:

`Learn how to install OpenNMS Horizon/Meridian in a Kubernetes or Red Hat OpenShift environment`

File: docs/modules/deployment/pages/core/containers.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+If your organization uses Kubernetes or Red Hat OpenShift, OpenNMS makes a Helm chart available to simplify {page-component-title} deployment.
+For information on deploying in a Docker environment, see the Docker tabs in https://docs.opennms.com/horizon/latest/deployment/core/getting-started.html[installation and configuration].
+
+Each deployment through Helm charts has a single Core server, Grafana, a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.
```
Comment (mershad-manesh) [inline docs/modules/deployment/pages/core/containers.adoc:9]: I don't believe we need this line
`Each deployment through Helm charts has a single Core server, Grafana, a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.`

File: docs/modules/deployment/pages/core/containers.adoc [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Horizon 32.x.
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+Meridian 2023.x.
+endif::[]
+
+== Before you begin
+
+You must have a deep understanding of containerized environments (https://kubernetes.io/docs/home/[Kubernetes], https://access.redhat.com/documentation/en-us/openshift_container_platform/[Red Hat OpenShift]) and https://helm.sh/docs[Helm], the package manager for Kubernetes.
```
Comment (mershad-manesh) [inline docs/modules/deployment/pages/core/containers.adoc:23]: we can remove `the package manager for Kubernetes.` ; technically Helm is the package manager

File: docs/modules/deployment/pages/core/containers.adoc [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+You should also be familiar with {page-component-title}, its components, and how the environment works.
+
+== Requirements
+
+* https://kubernetes.io/docs/reference/kubectl/[kubectl]
+* https://kubernetes.io/docs/reference/kubectl/[Helm] version 3
+* Kubernetes 1.20+ OR Red Hat OpenShift 4.12+
+
+When using cloud resources, make sure you have https://learn.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest[az] for Azure or https://cloud.google.com/sdk/gcloud[gcloud] for Google Cloud.
```
Comment (mershad-manesh) [inline docs/modules/deployment/pages/core/containers.adoc:33]: I think we can remove this; As this is going to depend on how the customer wants to manage their environment

**Review by mershad-manesh — CHANGES_REQUESTED** [review]
Overall looks good; there are few items that might need tweaking (see comments)

File: docs/modules/deployment/pages/core/containers.adoc (hunk 1 above)
Comment (Bonrob2) [reply docs/modules/deployment/pages/core/containers.adoc:4]: Yes. We have a variable I can use. I missed it here. Will update.

Comment (Bonrob2) [reply docs/modules/deployment/pages/core/containers.adoc:4]: Actually, this text is not included in the docs. It's descriptive text for search engines. Unfortunately, I can't use the variables here. But it never appears in the docs.

**Pull request merged**